lib/charms/mongodb/v0/helpers.py

```diff
@@ -0,0 +1,274 @@
+"""Simple functions, which can be used in both K8s and VM charms."""
+
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import json
+import logging
+import os
+import secrets
+import string
+import subprocess
+from typing import List, Optional, Union
+
+from charms.mongodb.v0.mongodb import MongoDBConfiguration, MongoDBConnection
+from ops.model import (
+    ActiveStatus,
+    BlockedStatus,
+    MaintenanceStatus,
+    StatusBase,
+    WaitingStatus,
+)
+from pymongo.errors import AutoReconnect, ServerSelectionTimeoutError
+
+from config import Config
+
+# The unique Charmhub library identifier, never change it
+LIBID = "b9a7fe0c38d8486a9d1ce94c27d4758e"
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version
+LIBPATCH = 9
```

Comment:
```
> charmcraft fetch-lib charms.mongodb.v0.helpers
Library charms.mongodb.v0.helpers version 0.7 downloaded.
```
I think the version here should be 8, I think charmhub will complain if you try to skip a version when publishing.